src/pages/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import Layout from '../components/Layout/Layout';
+import LandingSection from '../components/LandingSection';
+import ProjectsSection from '../components/ProjectsSection';
+import ContactSection from '../components/ContactSection';
+
+const aaa = () => {
```

Comment:
`aaa` ? Name is really shitty, but you *have to* give capitalized name for React component!